druid/src/env.rs

```diff
@@ -461,7 +461,14 @@ impl Data for EnvImpl {
 
 impl Default for Env {
     fn default() -> Self {
-        let l10n = L10nManager::new(vec!["builtin.ftl".into()], "./resources/i18n/");
+        Env::with_l10n(vec![])
```

Comment:
I think we should keep this as is, since it at least means that the examples continue to work. This all obviously needs to be reworked, though...

Reply:
Fixed